### PR TITLE
tutorials/keras/text_classification.ipynb: Refine exercise instruction

### PR DESCRIPTION
**Pull request by SuperKenVery**
Add an instruction to add a Softmax layer. Otherwise the accuracy would be around 0.3, not far away from the random-guess 0.25.

### CODE REVIEW

**Review by MarkDaoust — CHANGES_REQUESTED** [review]
Hi, thanks for pointing out this error.

Instead of adding a softmax layer, or softmax activation, could you set the loss function to:

`tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True)`

I'd edit your pr and merge it but it seems that you didn't check the "allow projecty maintainers to edit the PR" box.

Thanks!

**Review by SuperKenVery — COMMENTED** [review]
Change according to @MarkDaoust

**Review by SuperKenVery — COMMENTED** [review]
change according to @MarkDaoust

**Review by SuperKenVery — COMMENTED** [review]
change according to @MarkDaoust

**Review by SuperKenVery — COMMENTED** [review]
change according to @MarkDaoust

**Comment by SuperKenVery** [comment]
Well I'm not quite (actually quite not) familiar with GitHub and VSCode's GitHub plugin, so I accidentally added so many comments... Sorry. 

And I believe I didn't uncheck the box. 
image

**Comment by SuperKenVery** [comment]
image
I believe I checked the box. 

Actually it is checked by default and I didn't uncheck it, I remember.

**Review by MarkDaoust — APPROVED** [review]
Thanks!

**Review by 8bitmp3 — APPROVED** [review]
